src/rest_api.py

```diff
@@ -98,6 +100,17 @@ def sync_data():
         logger.info("Calling latest version sync with 'all'")
         _session.post(url, json=['all'])
         resp['message'] = resp['message'] + " for latest version"
+
+    cve_source_sync = input_json.get('cve_source_sync', False)
```

Comment:
Boolean default value doesnt go well with the json provided in the valid input section